### PR TITLE
ETCW-563: Release and deployment wiki for v3.15.2.

### PR DESCRIPTION
**Pull request by mtem27**
## JIRA
https://etcprojects.atlassian.net/browse/ETCW-563

## Description
Deployment of hotfix v3.15.2 to production.
Update wiki.

### CODE REVIEW

File: content/excelitas/releases.md [hunk 1]
```diff
@@ -7,6 +7,86 @@ next: '/excelitas/global/site-settings/'
...
+* **Type**:
+  - [ ] Normal release
+  - [x] Hotfix
+* No encountered error during deployment.
+
+### Tasks
+* Sprint 36
+  * [ETCW-927: Remove "Product Categories" from showing up in the list when "Remove from listing" is set]
+    (https://etcprojects.atlassian.net/browse/ETCW-927)
+  * [ETCW-928: Apply "Show in listings" field values functionality to display product and product category in "Sort" page]
```
Comment (timhtheos) [inline content/excelitas/releases.md:25]: Add another 2 spaces.

File: content/excelitas/releases.md [hunk 2]
```diff
@@ -7,6 +7,86 @@ next: '/excelitas/global/site-settings/'
...
+  (http://excelitas.prod.acquia-sites.com/AH_DOWNLOAD?t=1631619306&d=/mnt/gfs/excelitas.prod/backups/on-demand/backup-2021-09-14-11-11-excelitas-185182845.sql.gz&prod=384eca769b17022ac8b8b4127056380cf94722b45bb501677b6ec7fdab6c4b4b)
+
+On-demand database backups are user initiated, and are retained indefinitely,
+until manually deleted.
+
+### Code preparation
+
+* [ ] Updated local *develop* branch.
+* [x] Updated local *release-v3.15.2* branch.
+* [x] No issues and merge commits when *release-v3.15.1* branch was updated.
```
Comment (timhtheos) [inline content/excelitas/releases.md:43]: when *release-v3.15.2* branch was updated.

File: content/excelitas/releases.md (hunk 1 above)
Comment (mtem27) [reply content/excelitas/releases.md]: Updated, please check again.

File: content/excelitas/releases.md (hunk 2 above)
Comment (mtem27) [reply content/excelitas/releases.md]: Updated on the succeeding commit.

**Comment by timhtheos** [comment]
Checked in https://2e1ef1ab.excelitas-docs.pages.dev/excelitas/releases/#v3152---09142021

Merged.

**Pull request merged**